src/overlays/actors/ovl_En_Test5/z_en_test5.c

```diff
@@ -31,14 +28,85 @@ const ActorInit En_Test5_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void EnTest5_SetupAction(EnTest5* this, EnTest5ActionFunc actionFunc) {
+    this->actionFunc = actionFunc;
+}
+
+void EnTest5_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    EnTest5* this = THIS;
+    GlobalContext* globalCtx = globalCtx2;
+
+    WaterBox* water;
+    f32 _; // Unused water surface
```

Comment:
We put all the variables declarations together, without newlines in between. For the unused float, we prefer to use the function's parameter name when possible
```suggestion
    EnTest5* this = THIS;
    GlobalContext* globalCtx = globalCtx2;
    WaterBox* water;
    f32 ySurface; // Unused water surface
```